README.NewResourceFromTerraform.md

```diff
@@ -615,6 +615,11 @@ update to date, we need to add the new CRDs and samples to existing doc.
     `https://cloud.google.com` (i.e.
     `https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instances`).
     Add an additional paragraph/table if needed.
+1.  Update the URL in "Before you begin" section, points to the "Prerequisites" or 
+    "Before you begin" section in the service guide. Make sure that the URL
```

Comment:
From our public doc (e.g. https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containercluster), each word in the headers has its first letter capitalized.

I am not sure if it looks correct if we say "Before You Begin", but also I believe we want to keep the header format consistent. Could you check with Taryn about it? We can also just do "Prerequisites".

Reply:
Updated to use "Prerequisites"